packages/frontend-shared/src/components/Tooltip.vue

```diff
@@ -71,6 +71,17 @@ const actualPopperClass = computed(() => {
     result.push('no-arrow')
   }
 
+  // color takes priority, else fallback to tooltip -> dark, menu -> light
```

Comment:
Refactored so as to allow menu functionality styled with the dark tooltip theme.